topics/imaging/tutorials/yolo-segmentation-training/tutorial.md

```diff
@@ -0,0 +1,241 @@
+{: .hands_on}
+
+
+# Annotating the images
+Once the input images are preprocessed and converted to JPG format, we will need to manually annotate the images using AnyLabeling. AnyLabeling is an interactive labeling tool integrated into Galaxy that allows users to draw bounding boxes or segmentation shapes around objects of interest. 
+
+In this tutorial, we annotate all 22 example images using the tool's `Auto labeling` function, which helps accelerate the annotation process. The labels will later be used to train the YOLO model. The annotation process is entirely human-in-the-loop, so the quality and relevance of the labels directly influence the performance of the resulting model. 
+
+> <hands-on-title> Interactive Annotation </hands-on-title>
+>
+> 1. Start {% tool [AnyLabeling Interactive](interactive_tool_anylabeling) %} with the following parameters:
+>    - {% icon param-file %} *"Input images"*: `input-converted` (Input dataset collection, output of **Convert image format** {% icon tool %})
+>    - *"Pre-existing annotations"*: `Empty`
+>    - *"Classes file"*: `Empty`
+>    - *"A tarball containing custom model files and yaml files"*: `Empty`
+>
+>    {% snippet faqs/galaxy/interactive_tools_open.md  %}
+>
+> 2. Once AnyLabeling is open.
+     - Navigate to `File` -> `Change Output Dir` and set the output directory to `working/home/output`.
+> ![al change output dir](../../images/yolo-train/al_chng_dir.png){: width="75%"}
+>
+>    - To open your input images, go to `File` -> `Open Dir`, and choose `working/home/input_images` directory.
+>
+> ![al open dir](../../images/yolo-train/al_open_dir.png){: width="75%"}
+>
+>    - Click the `Auto Labeling` button, choose `Segment Anything 2 (Hiera-Tiny)` from the model dropdown, select `+Point`,then begin annotation by clicking on the object in the image. Wait for the inference to complete before continuing.
+>
+> ![al open dir](../../images/yolo-train/al_label.png){: width="75%"}
+>
+>    - If the inferred polygon is not accurate, click `-Point`, then click on a background (non-object) area to refine the results. The model will re-run the inference.
+>
+> ![al open dir](../../images/yolo-train/al_label_1.png){: width="75%"}
+>    - After the inference is complete, click `Finish Object`. In the `Enter object label` box, type `tail` then click `OK` to save the annotation.
+>
+> ![al open dir](../../images/yolo-train/al_finish.png){: width="75%"}
+>
+>    - **Important:** The object label must match the content of `class_names.txt`.
+>    - Repeat these steps for each image in the dataset to complete the interactive annotation process. 
+>
+{: .hands_on}
+
+
+# Prepare training data
+In this step, we will convert the annotation file generated by AnyLabeling (in JSON format) into YOLO compatible TXT files. This conversion is necessary because the YOLO training tool expects annotations in its specific text format, where each object is described by class ID and bounding box coordinates.
```

Comment:
```suggestion
In this step, we will convert the annotation file generated by AnyLabeling (in JSON format) into YOLO compatible TXT files. This conversion is necessary because the YOLO training tool expects annotations in its specific text format, where each object is described by a class ID and corresponding bounding box coordinates.
```